Collector/Twig/SlugifyExtension.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Http\HttplugBundle\Collector\Twig;
+
+/**
+ * @author Ibrahima SOW <sowbiba@hotmail.com>
+ */
+class SlugifyExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @return array
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('slugify', [$this, 'slugify'], ['is_safe' => ['html']]),
+        ];
+    }
+
+    /**
+     * @param string $string http message
```

Comment:
why http message? i think we can do without the phpdoc, or just `@param string $string` until we are on php 7 and can use scalar type hints.